examples/00-pyunits/basic_usage.py

```diff
@@ -83,8 +85,8 @@
 capacitance.units  # >>> "farad"
 capacitance.si_value  # >>> 50.0
 capacitance.si_units  # >>> "kg^-1 m^-2 s^4 A^2"
-capacitance.dimensions  # >>> [-1.0, -2.0, 4.0, 0.0, 0.0, 0.0, 0.0, 2.0, 0.0]
-capacitance.is_dimensionless  # >>> False
+capacitance.dimensions.dimensions  # >>> {0: -1.0, 1: -2.0, 2: 4.0, 8: 2.0}
```

Comment:
This usage is awkward (long object chain/trainwreck) and the output is meaningless with knowledge of the implementation. `capacitance.dimensions` itself contains the high-level information that is comprehensible.